Felienne/hedy
# Bulgarian translations

With this request we are adding a version of the programming course to Hedy in the Bulgarian language.


**Description**

Adds translations of levels 1 to 6 in Bulgarian. 
What is covered so far is the translations of the **main course** and **adventures** up to level 6.


**How to test**

To find and see the Bulgarian translations on the main course as well as on adventures across levels use the drop-down menu list for languages and hit the menu item "Български".
